### ISSUE TITLE
No matching distribution found for torch==1.3.0

### ISSUE THREAD

**Issue by ll7** [sisl/AdaptiveStressTestingToolbox#42]
Hello,

I got interested in the AST Toolbox after reading "A Survey of Algorithms for Black-Box Safety Validation" and tried to  `pip install ast-toolbox` in the docker image "jupyter/datascience-notebook:latest".

I received the following output:

```
Collecting ast-toolbox
  Using cached ast_toolbox-2020.9.1.0-py2.py3-none-any.whl (84 kB)
Requirement already satisfied: joblib in /opt/conda/lib/python3.8/site-packages (from ast-toolbox) (1.0.0)
Requirement already satisfied: cached-property in /opt/conda/lib/python3.8/site-packages (from ast-toolbox) (1.5.1)
Requirement already satisfied: python-dateutil in /opt/conda/lib/python3.8/site-packages (from ast-toolbox) (2.8.1)
Collecting garage==2019.10.1
  Using cached garage-2019.10.1.tar.gz (317 kB)
Requirement already satisfied: click in /opt/conda/lib/python3.8/site-packages (from garage==2019.10.1->ast-toolbox) (7.1.2)
Requirement already satisfied: cloudpickle in /opt/conda/lib/python3.8/site-packages (from garage==2019.10.1->ast-toolbox) (1.6.0)
Requirement already satisfied: matplotlib in /opt/conda/lib/python3.8/site-packages (from garage==2019.10.1->ast-toolbox) (3.3.3)
Requirement already satisfied: numpy>=1.14.5 in /opt/conda/lib/python3.8/site-packages (from garage==2019.10.1->ast-toolbox) (1.19.4)
Requirement already satisfied: psutil in /opt/conda/lib/python3.8/site-packages (from garage==2019.10.1->ast-toolbox) (5.8.0)
INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of ast-toolbox to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement torch==1.3.0 (from garage)
ERROR: No matching distribution found for torch==1.3.0```

Do you have any suggestions on how I could fix this?

Thanks for your work and kind regards.

**Comment by mark-koren** [comment]
Hello,

I need a bit more information on how you are handling the install process on that docker image. Please post the command you are using to start the docker image, as well as any custom docker launch or config files associated with the start-up or install process.

**Comment by ll7** [comment]
I did install docker and executed the following in the terminal:
`docker run -p 8888:8888 jupyter/datascience-notebook`
I opened the jupyter web interface in my browser and in the first cell I executed:
`!pip install ast-toolbox`
The output [above](https://github.com/sisl/AdaptiveStressTestingToolbox/issues/42#issue-783511690) was the result.

Nevertheless, I was able to install the ast-toolbox in colab. Therefore, this issue is neither urgent nor important and can be closed from my point of view.

Thank you.
